static/js/components/dialogs/hoc.js

```diff
@@ -0,0 +1,50 @@
+// @flow
+import React from 'react';
+import R from 'ramda';
+
+import * as commonUiActions from '../../actions/commonUI';
```

Comment:
Minor nitpick, feel free to ignore if you prefer: The filenames for commonUI and collectionsUI have a capitalized 'I' while videosUi has a lowercase 'i'

Reply:
I went back and forth mentally about whether I prefer `*Ui` or `*UI`, and I think I agree with you that `*Ui` is preferable. I'll change them